### DISCUSSION TITLE
Respect max length and break classes into new lines?

### DISCUSSION THREAD

**Discussion by omar2205** [tailwindlabs/tailwindcss#10283] [Ideas]
With how long tailwind classes can get, this plugin should break them to fix prettier's print width

**Comment by ony3000** [comment]
Hello! I am leaving a comment because it seems to be a similar topic to #7763.

There is a [prettier plugin](https://www.npmjs.com/package/prettier-plugin-classnames) that I released this year. Maybe this will help.

**Reply by omar2205** [reply]
looks good, cheers
